### PR TITLE
Fix/refactor responsive toolbar

### LINKED ISSUE #84: Refactor Hamburger Toolbar/Toolbar
Currently they are two separate components, which seems a bit unnecessary. Would be nice to refactor it one component that changes CSS based on size of screen

### PR DESCRIPTION
**Pull request by Fabricevladimir**
V2 features stripped. Waiting on final mobile designs. Closes #84

### CODE REVIEW

File: src/components/Toolbar/__tests__/Toolbar.test.jsx [hunk 1]
```diff
@@ -6,15 +6,21 @@ import { useUser } from '../../../contexts/UserContext';
...
+describe('Toolbar Component Tests', function() {
+  function setup(useUserValue = [null, null]) {
+    useUser.mockReturnValue(useUserValue);
+    const { queryAllByRole, queryByText, queryByTestId, debug } = renderWithRouter(
       <UserProvider>
         <Toolbar />
       </UserProvider>
     );
 
+    return { queryAllByRole, queryByText, queryByTestId, debug };
```
Comment (will-nemo) [inline src/components/Toolbar/__tests__/Toolbar.test.jsx]: damn if i knew i could do this i could make my tests duplicate less

File: src/components/Toolbar/__tests__/Toolbar.test.jsx [hunk 2]
```diff
@@ -6,15 +6,21 @@ import { useUser } from '../../../contexts/UserContext';
...
 const { UserProvider } = jest.requireActual('../../../contexts/UserContext');
 
-describe('Hamburger Toolbar Component Tests', function() {
-  test('should show navigation links', () => {
-    useUser.mockReturnValue([null, null]);
-    const { queryAllByRole, queryByText } = renderWithRouter(
+describe('Toolbar Component Tests', function() {
+  function setup(useUserValue = [null, null]) {
+    useUser.mockReturnValue(useUserValue);
+    const { queryAllByRole, queryByText, queryByTestId, debug } = renderWithRouter(
```
Comment (will-nemo) [inline src/components/Toolbar/__tests__/Toolbar.test.jsx]: Also i don't think queryByTestId and debug are ever used in these tests, if not I'd remove

File: src/style-sheets/layout/_toolbar.scss [hunk 3]
```diff
@@ -1,112 +1,135 @@
...
+    height: 100px;
 
-      &:hover {
-        opacity: 0.7;
+    & .icons {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+
+      & .logo {
```
Comment (will-nemo) [inline src/style-sheets/layout/_toolbar.scss]: I think you could shorten this nest to & .logo img {

**Comment by will-nemo** [comment]
This is sweet man, got rid of a lot of unnecessary code. I'm assuming the links are still centered on the navbar since there aren't any login buttons currently. If you could just make the color of those links #232936 and the font weight 600, to follow the mock ups, then this ticket should be close to merging

File: src/style-sheets/layout/_toolbar.scss (hunk 3 above)
Comment (Fabricevladimir) [reply src/style-sheets/layout/_toolbar.scss]: Yeah, I was expecting to have code specific to the `logo` class. It's pretty much unnecessary right now.

**Pull request merged**